docs/products/postgresql/howto/migrate-db-to-aiven-via-console.rst

```diff
@@ -0,0 +1,278 @@
+Migrate PostgreSQL® databases to Aiven using the console 
+========================================================
+
+You can migrate PostgreSQL® databases to Aiven using either CLI (``aiven-db-migrate``) or the Aiven web console. This article addresses the latter scenario by providing guidelines on how to use the Aiven console to migrate PostgreSQL databases to the Aiven platform.
+
+.. seealso::
+
+    For the other migration method (using ``aiven-db-migrate``), see :doc:`Migrate to Aiven for PostgreSQL® with aiven-db-migrate </docs/products/postgresql/howto/migrate-aiven-db-migrate>`.
+
+About migrating via console
+---------------------------
+
+* With the console migration tool, you can migrate existing on-premise or cloud-hosted PostgreSQL databases to clusters in your Aiven account.
+
+  .. note:: 
+    
+     The console migration tool doesn't support migrating managed database clusters on Aiven to other managed database clusters on Aiven.
+
+* Console migration tool uses `logical replication <https://www.postgresql.org/docs/current/logical-replication.html>`_ by default.
+
+* With the console migration tool, you can migrate PostgreSQL® databases either using the continuous migration method, which is the one recommended and detailed in this guide, or using the dump method (``pg_dump``).
+
+  * Continuous migration keeps the source database operational during the migration but requires superuser permissions.
```

Comment:
We use the same `aiven-db-migrate` under the hood for migration from the console, so the requirements should be the same, which means super user is needed or `aiven_extras` installed with probably replication permissions, I will check it and comment later.

Reply:
It does not require super user, just checked with avnadmin and installed aiven_extras, replication did work.

Reply:
@alexole , thank you - in that case I'm adding the use of installed aiven_extras as an alternative to superuser perms.